pkg/runtime/reconciler.go

```diff
@@ -36,10 +36,11 @@ import (
 // controller-runtime.Controller objects (each containing a single reconciler
 // object)s and sharing watch and informer queues across those controllers.
 type reconciler struct {
-	kc  client.Client
-	rmf acktypes.AWSResourceManagerFactory
-	rd  acktypes.AWSResourceDescriptor
-	log logr.Logger
+	kc        client.Client
+	rmf       acktypes.AWSResourceManagerFactory
+	rd        acktypes.AWSResourceDescriptor
+	log       logr.Logger
+	awsConfig ServiceControllerCloudCfg
```

Comment:
please call the field `cfg` and let's name the type just `Config` instead of `ServiceControllerCloudCfg`